src/slurm_plugin/clustermgtd.py

```diff
@@ -763,6 +767,7 @@ def _find_unhealthy_slurm_nodes(self, slurm_nodes):
                 consider_drain_as_unhealthy=self._config.terminate_drain_nodes,
                 consider_down_as_unhealthy=self._config.terminate_down_nodes,
                 log_warn_if_unhealthy=node.name not in reserved_nodenames,
+                ec2_backing_instance_timeout=self._config.ec2_backing_instance_timeout,
```

Comment:
there is no need to pass this all the way through the methods call up to where it's used.
In the method you need this, you can access it through `self._config.ec2_backing_instance_timeout`

Reply:
This node object doesn't have the value that is stored in cluster manager.  Is there a good way to add this variable to the SlurmNode object without resetting every time this loop runs?  I am sort of treating the slurm nodes as stateless because the state is reset every time through the loop